devops/openshift/queue-consumers-deploy.yml

```diff
@@ -120,6 +120,21 @@ objects:
                     secretKeyRef:
                       key: ${ZONE_B_SFTP_PRIVATE_KEY_NAME_KEY}
                       name: ${ZONE_B_SFTP_SECRET_NAME}
+                - name: GC_NOTIFY_URL
+                  valueFrom:
+                    secretKeyRef:
+                      key: ${GC_NOTIFY_URL_NAME_KEY}
+                      name: ${QUEUE_CONSUMERS_SECRET_NAME}
+                - name: GC_NOTIFY_API_KEY
+                  valueFrom:
+                    secretKeyRef:
+                      key: ${GC_NOTIFY_API_KEY_NAME_KEY}
+                      name: ${QUEUE_CONSUMERS_SECRET_NAME}
+                - name: GC_NOTIFY_TO_ADDRESS
+                  valueFrom:
+                    secretKeyRef:
+                      key: ${GC_NOTIFY_TO_ADDRESS_NAME_KEY}
+                      name: ${QUEUE_CONSUMERS_SECRET_NAME}
```

Comment:
As the queue secrets are growing and is not shared, lets discuss if we need separate secrets for queues during our dev sync up.